server/test/stringTests.ts

```diff
@@ -0,0 +1,44 @@
+import { upperCamel, lowerSnake } from '../common';
+// Import * as mocha from "mocha";
+// Import { expect } from "chai";
+
+// All graphql requests are formatted lower_snake (readability)
+// All internal configs are upper camel (matching filename convention)
+// Test the functions that do these conversions
+const testUpperCamel = (before: string, after: string) => {
+	const actual = upperCamel(before);
+	console.log(`Given "${before}", upperCameled to "${actual}".` );
+	if (actual !== after) console.log(`ERROR: expected "${after}".`);
+	return actual === after;
+};
+
+// Converts camel and snake to lower_snake (for plugin)
+const testLowerSnake = (before: string, after: string) => {
+	const actual = lowerSnake(before);
+	console.log(`Given "${before}", upperCameled to "${actual}".` );
+	if (actual !== after) console.log(`ERROR: expected "${after}".`);
+	return actual === after;
+};
+
+function main() {
+	console.log("Running tests...");
+	// Upper camel has less worry about edge cases since graphql is machine generated
+	testUpperCamel("", "");
+	testUpperCamel("live", "Live");
+	testUpperCamel("LiveSite", "LiveSite");
```

Comment:
When you use expect and other fancy tools this should just be like:
```typescript
upperCamel("LiveSite").equals("LiveSite");
```
or something similar (I don't remember the syntax).